src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -204,9 +350,9 @@ public function exchange(): \AMQPExchange
         return $this->amqpExchange;
     }
 
-    public function getConnectionCredentials(): array
+    public function getConnectionConfiguration(): array
```

Comment:
That's a BC break, but the component is experimental.
Still, should be mentioned in the upgrade/changelog files I suppose?

Reply:
Yes, we can definitely mention this in the upgrade file!

Reply:
@nicolas-grekas updated the CHANGELOG 👍